install/kubernetes/cilium/charts/operator/templates/clusterrole.yaml

```diff
@@ -57,11 +57,31 @@ rules:
   - get
   - list
   - watch
+# For cilium-operator running in HA mode.
```

Comment:
I think we decided to drop the support for the `ConfigMapLock` and not support cilium operator HA in k8s <= 1.14?